### PR TITLE
Updated method browser.get() to use baseURL if no parameters passed

### PR DESCRIPTION
**Pull request by Xotabu4**
Updated method browser.get() to use baseURL if no parameters passed
Added test for default destination parameter

### CODE REVIEW

**Comment by Xotabu4** [comment]
I signed it!

**Comment by Xotabu4** [comment]
Is there a reason to use old selenium server 3.0.4-beta4.jar?

Getting multiple WebDriverErrors on attempt to start browser on CircleCI only. That might be caused by incompatible Selenium Server<->Chrome Driver<->Chrome browser versions
```
[08:33:04] E/launcher - Error: WebDriverError: no such session
  (Driver info: chromedriver=2.28.455506 (18f6627e265f442aeec9b6661a49fe819aeeea1f),platform=Linux 3.13.0-107-generic x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 3 milliseconds
Build info: version: '3.0.0-beta4', revision: '3169782', time: '2016-09-29 10:29:23 -0700'
System info: host: 'box59', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.13.0-107-generic', java.version: '1.8.0_102'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{message=unknown error: Chrome version must be >= 55.0.2883.0
  (Driver info: chromedriver=2.28.455506 (18f6627e265f442aeec9b6661a49fe819aeeea1f),platform=Linux 3.13.0-107-generic x86_64), platform=ANY}]
```


I believe thats the reason why it might fail.

Any other thoughts?

File: lib/browser.ts [hunk 1]
```diff
@@ -851,11 +851,11 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * browser.get('https://angularjs.org/');
    * expect(browser.getCurrentUrl()).toBe('https://angularjs.org/');
    *
-   * @param {string} destination Destination URL.
+   * @param {string=} opt_destination Destination URL. Uses 'config.baseUrl' by default
```
Comment (heathkit) [inline lib/browser.ts]: We're on TypeScript now, please remove typings from @param. Leaving the description is fine, but only if it's meaningful. ("@param opt_destination Destination URL" is not a meaningful comment, but mentioning the default is)

**Comment by heathkit** [comment]
Sorry, the CI was broken. Please rebase this and we'll take another look.

File: lib/browser.ts (hunk 1 above)
Comment (Xotabu4) [inline lib/browser.ts]: Thanks!
I see typings in comments everywhere in this file and project. Should this be fixed globally? 

I am not sure how to better write this line, could you help?

**Comment by vladmm** [comment]
Hi all, 

What is the status for this PR? Is someone still taking care of it?

Cheers

**Pull request closed without merging**